changelog.md

```diff
@@ -29,6 +31,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - \[Demo\] Screens in about section now use current theme colors ([#390](https://github.com/Orange-OpenSource/ods-android/issues/390))
 - \[Lib\] Fix cards vocalization when clickable ([#391](https://github.com/Orange-OpenSource/ods-android/issues/391))
 
+### Removed
+
+- \[Lib\] `OdsOutlinedTextfield` and `OdsPasswordOutlinedTextField` have been removed and replaced by `OdsTextField` and `OdsPasswordTextField`. Text fields appearance (outlined or filled) is now managed by the theme configuration. ([#415](https://github.com/Orange-OpenSource/ods-android/issues/415))
```

Comment:
I don't know if we need to keep two separate entries for this. Maybe we could just replace `[Lib]` by `[All]` and remove the one in `changed` section.